NBitcoin.Miniscript.Tests/CSharp/MiniscriptPSBTTests.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using NBitcoin.Miniscript;
+using NBitcoin;
+using Xunit;
+namespace NBitcoin.Tests
+{
+	public class MiniscriptPSBTTests
+	{
+		public MiniscriptPSBTTests()
+		{
+		}
+
+        private TransactionSignature DummyKeyFn(PubKey key) => null;
+        public void ShouldSatisfyMiniscript()
+		{
```

Comment:
FYI mixed tabs and spaces, misconfigured editor? ;)

Reply:
😭
I will fix later.